### DISCUSSION TITLE
RPC request GetLlmToken failed: account to young

### DISCUSSION THREAD

**Discussion by flamaso** [zed-industries/zed#19330] [Feature Requests]
I installed zed , As I was trying to use the AI model, this problem occurred: 

“RPC requesr GetLlmToken failed : account to young

**Comment by hideonebush** [comment]
how young is too young please ?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
